### DISCUSSION TITLE
Is `jekyll-data` still in use?

### DISCUSSION THREAD

**Discussion by imba-tjd** [mmistakes/minimal-mistakes#2579] [Q&A] [answered]
I see `jekyll-data` is added in https://github.com/mmistakes/minimal-mistakes/commit/9c318fc3f1842382ae6913a9dea7b4af1714b587 and removed in https://github.com/mmistakes/minimal-mistakes/commit/2388cbe020e7edab658730f998be11e04af711d3, but the CHANGELOG and documentation didn't say anything about the removal.

If it's valid, I wonder what settings would be read? Will it read settings in https://github.com/mmistakes/minimal-mistakes/blob/master/_config.yml?

**Comment by mmistakes** [comment, marked as answer]
It was removed. No settings are read from the bundled theme gem’s config or data files.

You’ll want to set those locally in your repo as one would normally do with a Jekyll site.

**Comment by imba-tjd** [comment]
Then I think the doc needs updating

**Comment by mmistakes** [comment]
PRs are welcome if you want to update.
